gm4_double_doors/README.md

```diff
@@ -0,0 +1,9 @@
+# Double Doors<!--$headerTitle--><!--$pmc:delete-->
+
```

Comment:
I would *love* it if you could make a quick screenshot of the "3-tall" doors for here. (Or, even better, a GIF of the opening if you have time)

Reply:
A gif for this is in the works with @Dennis-0. ETA is next week!